docs/Installing-the-project.md

```diff
@@ -17,6 +17,14 @@ Add the following to pom.xml:
 </dependency>
 ```
 
+If you haven't already, change the Java version:
+```
+<properties>
+    <maven.compiler.source>11</maven.compiler.source>
```

Comment:
the minimum supported one is actually 1.8

Reply:
alright, i'll change it to 1.8